databricks/koalas/series.py

```diff
@@ -4258,6 +4258,60 @@ def xs(self, key, level=None):
 
         return _col(DataFrame(internal))
 
+    def pct_change(self, periods=1):
+        """
+        Percentage change between the current and a prior element.
+
+        .. note:: the current implementation of this API uses Spark's Window without
+            specifying partition specification. This leads to move all data into
+            single partition in single machine and could cause serious
+            performance degradation. Avoid this method against very large dataset.
+
+        Parameters
+        ----------
+        periods : int, default 1
+            Periods to shift for forming percent change.
+
+        Returns
+        -------
+        Series
+
+        Examples
+        --------
+
+        >>> kser = ks.Series([90, 91, 85])
+        >>> kser
+        0    90
+        1    91
+        2    85
+        Name: 0, dtype: int64
+
+        >>> kser.pct_change()
+        0         NaN
+        1    0.011111
+        2   -0.065934
+        Name: 0, dtype: float64
+
+        >>> kser.pct_change(periods=2)
+        0         NaN
+        1         NaN
+        2   -0.055556
+        Name: 0, dtype: float64
+        """
+        sdf = self._internal._sdf
+        window = Window.orderBy(self._internal.index_columns).rowsBetween(-periods, -periods)
```

Comment:
`index_scols` instead of `index_columns`?

Reply:
Btw, seems like we should order by `F.monotonically_increasing_id()` instead of index?

```py
>>> import pandas as pd
>>> pser = pd.Series([90, 91, 85], index=[2,4,1])
>>> pser
2    90
4    91
1    85
dtype: int64
>>> pser.pct_change()
2         NaN
4    0.011111
1   -0.065934
dtype: float64
>>> pser.sort_index().pct_change()
1         NaN
2    0.058824
4    0.011111
dtype: float64

>>> ks.from_pandas(pser).pct_change()
1         NaN
2    0.058824
4    0.011111
Name: 0, dtype: float64
```